src/Propel/Common/Config/Loader/FileLoader.php

```diff
@@ -61,6 +61,8 @@ public function __construct(FileLocatorInterface $locator = null)
      * Replaces parameter placeholders (%name%) by their values for all parameters.
      *
      * @param array $configuration The configuration array to resolve
+     *
+     * @return array|void
```

Comment:
PHP7 has a void return type in its core, so it's semantically helpful to say this function returns an array or simply something insignificant. Array or null is something different and would imply that null is significant where `void` says pretty much the opposite - nothing new for you.

Since this function does not actually return anything, `void` is perfectly correctly placed here. 

A question that can be asked later is why this function returns void but that's out of scope of this PR. This PR is about making the doc and other stuff coherent with coding standard and semantically correct annotations. This annotation is correctly placed so should stay that way. Changing that now means changing the purpose of this PR - which should be avoided at all costs, considering its huge diff already.